pyresample/geometry.py

```diff
@@ -391,6 +391,28 @@ def get_edge_bbox_in_projection_coordinates(self, frequency: Optional[int] = Non
         x, y = self._get_bbox_elements(self.get_proj_coords, frequency)
         return np.hstack(x), np.hstack(y)
 
+    def boundary(self, frequency=None, force_clockwise=False):
```

Comment:
Thoughts on `to_boundary`?

Reply:
Yes. Here is how I planned to design the access to the new spherical interface: 
```
swath_def.boundary().polygon()    # SPolygon
swath_def.boundary().line()           # SLine
swath_def.boundary().polygon().to_shapely()  # shapely Polygon
swath_def.boundary().line().to_shapely()         # shapely Line
```
Would you prefer the following pattern?
```
swath_def.to_boundary().to_polygon().to_shapely()
swath_def.to_boundary().to_line().to_shapely()
```

Personally I prefer the first option .. it's more "readable"

Reply:
Sure. There seems like a law of Demeter violation, but maybe I don't fully understand that concept either.

Leave it as is.